airbyte-integrations/connectors/source-looker/unit_tests/test_url.py

```diff
@@ -0,0 +1,46 @@
+import re
+import pytest
+
+def valid_url(test_str):
+    pattern="(^[a-zA-Z0-9._-]*\.looker\.com$)|(^(?:[a-zA-Z0-9._-]*)?[a-zA-Z0-9._-]*\.[a-zA-Z0-9._-]*(?::\d{1,5})?$)|(^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}(?::\d{1,5})?$)"
+    result = re.match(pattern, test_str)
+    if result is not None:
+        return result.group(0)
+
+@pytest.fixture
+def url():

```

Comment:
this is not what I mean, there is an example of parameterized tests on the page that I sent.